JustSaying.Sample.Restaurant.KitchenConsole/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "System": "Warning",
+      "Microsoft": "Warning"
+    }
+  },
+  "AllowedHosts": "*"
```

Comment:
I guess this setting is redundant for a console app?

Reply:
yeah it was a copy/paste error - done